internal/testing/e2e/e2e

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+﻿#!/opt/homebrew/opt/bash/bin/bash
```

Comment:
This should probably be `#!/bin/env bash` to be OS-agnostic.

Reply:
Thanks! I made that change for testing and forgot to revert it.